File: app/src/main/java/me/imzack/app/end/view/fragment/ProblemsFragment.java
package me.imzack.app.end.view.fragment;

import android.os.Bundle;
import android.support.v7.widget.RecyclerView;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import me.imzack.app.end.R;
import me.imzack.app.end.view.adapter.ProblemListAdapter;

import butterknife.BindView;
import butterknife.ButterKnife;

public class ProblemsFragment extends BaseFragment {

    @BindView(R.id.list_problem)
    RecyclerView mProblemList;

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_problems, container, false);
    }

    @Override
    public void onViewCreated(View view, Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        ButterKnife.bind(this, view);

        mProblemList.setAdapter(new ProblemListAdapter());
    }

    @Override
    public void onDetach() {
        super.onDetach();
    }
}

File: app/src/main/kotlin/me/imzack/app/end/view/fragment/ProblemsFragment.kt
package me.imzack.app.end.view.fragment

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import kotlinx.android.synthetic.main.fragment_problems.*
import me.imzack.app.end.R
import me.imzack.app.end.view.adapter.ProblemListAdapter

class ProblemsFragment : BaseFragment() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?) =
            inflater.inflate(R.layout.fragment_problems, container, false)!!

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

        list_problem.adapter = ProblemListAdapter()
    }

    override fun onDetach() {
        super.onDetach()
    }
}
